docs/end-to-end-demo.md

```diff
@@ -45,6 +45,16 @@ This demo will demonstrate end to end Akri flow, all the way from discovering lo
     ```sh
     kubectl label node ${HOSTNAME,,} node-role.kubernetes.io/master= --overwrite=true
     ```
+1. Akri depends on crictl to track some Pod information and becase Microk8s does not install crictl locally, crictl must be installed and the Akri Helm chart needs to be configured with the crictl path and Microk8s containerd socket
```

Comment:
Bit of a run on. How about: "Akri depends on crictl to track some Pod information. MicroK8s does not install crictl locally, so crictl must be installed and the Akri Helm chart needs to be configured with the crictl path and the Microk8s containerd socket.